3p/iframe-messaging-client.js

```diff
@@ -100,6 +100,17 @@ export class IframeMessagingClient {
    */
   setupEventListener_() {
     listen(this.win_, 'message', event => {
+      try {
+        if (!this.hostWindow_.postMessage && this.hostWindow_.contentWindow) {
```

Comment:
That's not what I'm trying to do here. I'm trying to determine whether this.hostWindow_ is a window or a frame. If it's a frame, it won't have .postMessage(), it'll have .contentWindow.postMessage(). I've split it into a helper method to make this clearer.